sdk/src/connection.ts

```diff
@@ -0,0 +1,75 @@
+import {
+	Commitment,
+	GetMultipleAccountsConfig,
+	PublicKey,
+	RpcResponseAndContext,
+	AccountInfo,
+} from '@solana/web3.js';
+import { promiseTimeout } from './util/promiseTimeout';
+
+// interface for solana Connection
+interface SolanaConnection {
+	rpcEndpoint: string;
+
+	getMultipleAccountsInfoAndContext(
```

Comment:
do you have any idea of how this will work with web sockets?

Reply:
`Connection` requires both http and ws endpoints, only certain subscription methods are available over ws. I believe these polling based methods should be used even if a ws endpoint is provided.

i.e. `getMultipleAccountsInfoAndContext` uses `_rpcRequest` https://github.com/solana-labs/solana-web3.js/blob/master/src/connection.ts#L2934

 and all the [notification streams](https://docs.solana.com/developing/clients/jsonrpc-api#subscription-websocket) use `_rpcWebsocket` https://github.com/solana-labs/solana-web3.js/blob/master/src/connection.ts#L2533


which leads me to think that even if a user configures `ClearingHouse` to use websocket, they might still need to be running a `BulkAccountLoader`? otherwise risk missing messages